test/functional/developer_portal/admin/account/braintree_blue_controller_test.rb

```diff
@@ -18,7 +18,7 @@ class DeveloperPortal::Admin::Account::BraintreeBlueControllerTest < DeveloperPo
 
     @controller.stubs(current_account: @account)
 
-    get :hosted_success
+    get :hosted_success, form_params
```

Comment:
it works for `GET` and `POST` but probably better with `POST`

Reply:
It is about storing credit card data so a `POST` makes more sense to me than a `GET`, but anyway I am aware that this PR is just to make it word and thinking in improving it later.
Also we are deprecating functional tests and making integration tests instead, but I know that you said:

> I will do better integration tests later